rust/operator-binary/src/controller.rs

```diff
@@ -785,8 +833,13 @@ fn add_s3_volume_and_volume_mounts(
 ) -> Result<()> {
     if let Some(s3_conn) = s3_conn {
         if let Some(credentials) = &s3_conn.credentials {
-            pb.add_volume(credentials.to_volume("s3-credentials"));
-            cb_druid.add_volume_mount("s3-credentials", STACKABLE_SECRETS_DIR);
+            let volume_name = "s3-credentials";
```

Comment:
We might even consider to move this into a `const`. What is your opinion on that?

Reply:
haha, I had that exact thought at the time of writing. But thought it's a negligible performance improvement (and technically negligible size increase :rofl:).

Given it has been mentioned, I'll do it :smile:

Reply:
It does mean that the error context now looks like:

```rust
                CredentialsToVolumeSnafu {
                    volume_name: VOLUME_NAME,
                },
```

Reply:
Resolved in a8b2bc349f8cf5ab02e7dd8e16de17f6c888e63c